openapi/openapi.yaml

```
@@ -1327,6 +1327,8 @@ paths:
     $ref: paths/storefront/invoices.yaml
   '/storefront/invoices/{id}':
     $ref: paths/storefront/invoices@{id}.yaml
+  '/storefront/upcoming-invoices/{id}':
```

```suggestion
  '/storefront/orders/{id}/upcoming-invoice':
```

I am wondering why it is different from the Core?

---

Understood. Thanks!